apps/src/blockly/addons/contextMenu.js

```diff
@@ -205,6 +205,69 @@ const registerKeyboardNavigation = function() {
   GoogleBlockly.ContextMenuRegistry.registry.register(keyboardNavigationOption);
 };
 
+/**
+ * Change workspace theme to classic CdoTheme
+ */
+const registerCdoTheme = function() {
+  const cdoThemeOption = {
+    displayText: function(scope) {
+      return (
+        (isCurrentTheme('classic', scope.workspace)
+          ? '✓ '
+          : `${msg.enable()} `) + msg.blocklyClassicTheme()
+      );
+    },
+    preconditionFn: function(scope) {
+      if (isMusicLabTheme(scope.workspace)) {
+        return 'hidden';
+      } else if (isCurrentTheme('classic', scope.workspace)) {
```

Comment:
can we add constants for these strings? (`classic`, `disabled`, etc.)

Reply:
I don't think we can, or at least I'm not sure how after about 45 minutes of trying. The problem seems to be getting the imported constant to be in scope. These functions have specific parameters that we don't control:
https://github.com/google/blockly/blob/4125fd31534dd9f450219259b9ccb3c1495605c9/core/contextmenu_registry.ts#L137-L144

So far, anything I've tried has caused the constant to be undefined, unless I defined within the scope of this `preconditionFn` function.

Reply:
can we not declare them at the top of this file?
`const hidden = 'hidden'`

Reply:
Thanks - I got that to work. I think I was trying something overly complicated yesterday. Let me know what you think: https://github.com/code-dot-org/code-dot-org/pull/50264/commits/77b42ad817f75af6c1ac3b7bfd60ddafe205921b